src/main/java/org/mongoflink/sink/MongoCommitter.java

```diff
@@ -24,11 +24,14 @@
  **/
 public class MongoCommitter implements Committer<DocumentBulk> {
```

Comment:
These changes look like reverting #8. Can we exclude that?

Reply:
done